src/symbol/source.rs

```diff
@@ -78,3 +81,155 @@ impl super::Dwarf {
             .to_string())
     }
 }
+
+pub type CrabResult<T> = Result<T, Box<dyn std::error::Error>>;
+
+#[derive(Debug)]
+/// A line in a source code is represented as a line number and the string.
```

Comment:
```suggestion
/// A line in a source code is represented as a line number and the string.
#[derive(Debug)]
```

Doc comments are often written before the attributes of a type.